app/controllers/job_applications_controller.rb

```diff
@@ -1,4 +1,4 @@
-class AppliesController < ApplicationController
+class job_applicationsController < ApplicationController
```

Comment:
class names cannot have underscore, please change it. Follow the convention.

Reply:
the class name is changed